cmake/modules/Findkeyutils.cmake

```diff
@@ -13,7 +13,7 @@ INCLUDE(CheckCXXSymbolExists)
 
 # include dir
 
-find_path(KEYUTILS_INCLUDE_DIR keyutils.h NO_DEFAULT_PATH PATHS
+find_path(KEYUTILS_INCLUDE_DIR keyutils.h PATHS
```

Comment:
@tchaikov I update the PR however I was not able to test it as I don't have access to the machine I did the build at the moment.